tox.ini

```diff
@@ -1,13 +1,16 @@
 [tox]
 envlist =
     py{38,39,310}-django3.2,
-    py{38,39,310,311}-django{4.1,4.2},
+    py{38,39,310,311}-django4.1,
+    py{38,39,310,311,312}-django4.2,
```

Comment:
[Django 4.2.8 added Python 3.12 support](https://docs.djangoproject.com/en/4.2/releases/4.2.8/), so I suppose we should test this combination too now.